apps/dashboard/app/(app)/workspace-navigations.tsx

```diff
@@ -91,22 +90,31 @@ export const createWorkspaceNavigation = (
       href: "/monitors/verifications",
       label: "Monitors",
       active: segments.at(0) === "verifications",
-      hidden: !workspace.features.webhooks,
+      hidden: getFlag(workspace, "webhooks", {
```

Comment:
that's reversed too
it's too confusing this way I think
in the db I'd set it to `{ "webhooks": true }` if I want to give someone access, but now that would revoke access...